From 6cfabc91aef93de39d390f3148b6f339dbb4166a Mon Sep 17 00:00:00 2001
From: Roman Dobrodii <rdobrodii@antmicro.com>
Date: Wed, 8 Mar 2023 17:53:01 +0100
Subject: [PATCH 6/8] nvidia: remove obsolete dt-bindings/leds

This one in nvidia/ directory overrides the usual one in include/.
The one in nvidia/ is old, so modern DTS which use e.g.
LED_COLOR_* do not compile.

Signed-off-by: Roman Dobrodii <rdobrodii@antmicro.com>
---
 .../kernel-include/dt-bindings/leds/common.h  | 21 -------------------
 .../dt-bindings/leds/leds-netxbig.h           | 18 ----------------
 .../dt-bindings/leds/leds-ns2.h               |  8 -------
 3 files changed, 47 deletions(-)
 delete mode 100644 nvidia/soc/tegra/kernel-include/dt-bindings/leds/common.h
 delete mode 100644 nvidia/soc/tegra/kernel-include/dt-bindings/leds/leds-netxbig.h
 delete mode 100644 nvidia/soc/tegra/kernel-include/dt-bindings/leds/leds-ns2.h

diff --git a/nvidia/soc/tegra/kernel-include/dt-bindings/leds/common.h b/nvidia/soc/tegra/kernel-include/dt-bindings/leds/common.h
deleted file mode 100644
index 79fcef72ef57..000000000000
--- a/nvidia/soc/tegra/kernel-include/dt-bindings/leds/common.h
+++ /dev/null
@@ -1,21 +0,0 @@
-/*
- * This header provides macros for the common LEDs device tree bindings.
- *
- * Copyright (C) 2015, Samsung Electronics Co., Ltd.
- *
- * Author: Jacek Anaszewski <j.anaszewski@samsung.com>
- */
-
-#ifndef __DT_BINDINGS_LEDS_H__
-#define __DT_BINDINGS_LEDS_H
-
-/* External trigger type */
-#define LEDS_TRIG_TYPE_EDGE	0
-#define LEDS_TRIG_TYPE_LEVEL	1
-
-/* Boost modes */
-#define LEDS_BOOST_OFF		0
-#define LEDS_BOOST_ADAPTIVE	1
-#define LEDS_BOOST_FIXED	2
-
-#endif /* __DT_BINDINGS_LEDS_H */
diff --git a/nvidia/soc/tegra/kernel-include/dt-bindings/leds/leds-netxbig.h b/nvidia/soc/tegra/kernel-include/dt-bindings/leds/leds-netxbig.h
deleted file mode 100644
index 92658b0310b2..000000000000
--- a/nvidia/soc/tegra/kernel-include/dt-bindings/leds/leds-netxbig.h
+++ /dev/null
@@ -1,18 +0,0 @@
-/*
- * This header provides constants for netxbig LED bindings.
- *
- * This file is licensed under the terms of the GNU General Public
- * License version 2.  This program is licensed "as is" without any
- * warranty of any kind, whether express or implied.
- */
-
-#ifndef _DT_BINDINGS_LEDS_NETXBIG_H
-#define _DT_BINDINGS_LEDS_NETXBIG_H
-
-#define NETXBIG_LED_OFF		0
-#define NETXBIG_LED_ON		1
-#define NETXBIG_LED_SATA	2
-#define NETXBIG_LED_TIMER1	3
-#define NETXBIG_LED_TIMER2	4
-
-#endif /* _DT_BINDINGS_LEDS_NETXBIG_H */
diff --git a/nvidia/soc/tegra/kernel-include/dt-bindings/leds/leds-ns2.h b/nvidia/soc/tegra/kernel-include/dt-bindings/leds/leds-ns2.h
deleted file mode 100644
index 491c5f974a92..000000000000
--- a/nvidia/soc/tegra/kernel-include/dt-bindings/leds/leds-ns2.h
+++ /dev/null
@@ -1,8 +0,0 @@
-#ifndef _DT_BINDINGS_LEDS_NS2_H
-#define _DT_BINDINGS_LEDS_NS2_H
-
-#define NS_V2_LED_OFF	0
-#define NS_V2_LED_ON	1
-#define NS_V2_LED_SATA	2
-
-#endif
-- 
2.25.1

